### DISCUSSION TITLE
Release 5.28.1

### DISCUSSION THREAD

**Discussion by legobeat** [verdaccio/verdaccio#4237] [Q&A] [answered]
Would be nice to get a release with the dependency fixes and updates merged since `5.27.0`.

~376eb5378 could affect users who extend the Docker image but that lives outside the npm package itself and otherwise it looks like all non-breaking maintenance changes.~

**Comment by juanpicado** [comment]
What do you mean extacly, do you refer to docker image only?, that commit is from master, v5 lives on branch 5.x.

**Comment by legobeat** [comment]
> What do you mean extacly, do you refer to docker image only?, that commit is from master, v5 lives on branch 5.x.

Oh, I was hoping for a release including all the maintenance dependency-updates being published on `npmjs.org`. The docker update is not related to this.

I see `v5.28.0` has just been released, so updating this issue accordingly.

**Comment by legobeat** [comment]
I realized now that you're releasing production releases from `v5.x`  while `master` seems to be more of a dev-branch - so I guess fixes merged on `master` will have to be backported as well?

Is there a timeline/intention for releasing `6.0.0` at some point?

**Comment by juanpicado** [comment, marked as answer]
Yes on branch 6.x, there is a discussion about it you can check why. Both branches benefit highly of any update from  master (observe dependencies). Whenever 5.x is out of support 6.x will move to master and the idea is have one single branch, unfortunately I am not dedicated 100% and the world moves faster than my contributions :) so is some sort of workaround to make my life easier.
